internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -95,7 +95,7 @@ func (ksd KubernetesServiceDeployer) SetUp(ctxt ServiceContext) (DeployedService
 
 	ctxt.Name = kind.ControlPlaneContainerName
 	ctxt.Hostname = kind.ControlPlaneContainerName
-	ctxt.Agent.Host.NamePrefix = "kind-fleet-agent-"
+	ctxt.Agent.Host.NamePrefix = "kind-control-plane"
```

Comment:
nit: missing trailing dash or is it on purpose?

I'm not sure about implications of this change. Would you mind sharing more context? Will it affect any existing integrations